doc/docs.md

```diff
@@ -130,12 +128,13 @@ age = 21
 println(age)
 ```
 
-To change the value of the variable use `=`. In V, variables are
+To change the value of the variable, use `=`. In V, variables are
 immutable by default. To be able to change the value of the variable, you have to declare it with `mut`.
 
 Try compiling the program above after removing `mut` from the first line.
 
-Please note the difference between `:=` and `=`  
+Please note the difference between `:=` and `=`:
```

Comment:
The colon sign at the end is slightly confusing since the line discusses also syntax elements with colons in them.
It can be reworded as:
```
Please, note that there is a difference between the `:=` and `=` .
+ `:=` is used for declaring and initializing
+ `=` is used for assigning
```